Makefile

```diff
@@ -173,6 +173,8 @@ endif
 ## gatewayapi-provider
 GATEWAYAPI_PROVIDER ?= istio
 
+WITH_EXTENSIONS ?= false
```

Comment:
Instead of a boolean, maybe we could use a list of extensions, which gets fed into the extension manager as the set to start? if list is empty disabled?